spec/features/users/new_user_page_spec.rb

```diff
@@ -0,0 +1,38 @@
+feature 'user sign\'s up' do
```

Comment:
To avoid this dirty char escaping you can use `%()`

`feature %(user sign's up) do`